CSS/style.css

```diff
@@ -0,0 +1,119 @@
+#navbar {
+  background: rgb(216, 30, 30);
+  font-size: 20px;
+  margin: 0;
+  padding: 0;
+  height: 50px;
+  width: 98.8%;
+  position: fixed;
+  top: 0;
+}
+
+#navbar ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+}
+
+li {
+  float: right;
+}
+
+li a {
+  display: block;
+  padding: 20px;
+  color: black;
+  text-decoration: none;
+  font-size: 15px;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+
+li a:hover {
+  background-color: aquamarine;
+  height: 12px;
+}
+
+#main {
+  background-color: black;
+  color: white;
+  text-align: center;
+  padding: 20%;
+  font-size: 50px;
+}
+
+#main2 {
+  background-color: aqua;
+  color: red;
+  padding: 20%;
+}
+
+#main2 h1 {
+  font-size: 50px;
+  top: 1px;
+  text-align: center;
+}
+
+.column {
+  float: left;
+  width: 33.33%;
+  padding: 5px;
+}
+
+/* Clear floats after image containers */
+.row::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+
+#button1 {
+  background-color: black;
+  color: white;
+  float: left;
+  width: 33.33%;
+  padding: 5px;
+}
+
+#button2 {
+  background-color: black;
+  color: white;
+  float: left;
+  width: 33.33%;
+  padding: 5px;
+}
+
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+}
+
+#main3 {
+  background-color: rgb(32, 37, 37);
+  color: red;
+  padding: 10%;
```

Comment:
Please use pixels instead of percentages.

Reply:
Awesome! Can you please state the reason(s) for the required changes? It may be obvious to you and not so for others. 👍